challenge_8/python/mjuiuc/test.py

```diff
@@ -0,0 +1,93 @@
+import unittest
+import random
+import time
+from listNode import listNode
+from solution import solution
+
+class Tests(unittest.TestCase):
```

Comment:
I know this is just test code, but did you copy and paste `test1` to `test2` and `test3`? If you find your self pressing cntrl+c, then it's probably a good opportunity for abstraction.  Also, what do these tests do? You should have python docs that describe what the test is and expected value, ideally your test methods should reflect what you're testing.